core/domain/suggestion_services.py

```diff
@@ -469,7 +469,7 @@ def reject_suggestions(suggestion_ids, reviewer_id, review_message):
 
     feedback_services.create_messages(
         suggestion_ids, reviewer_id, feedback_models.STATUS_CHOICES_IGNORED,
-        None, review_message
+        None, review_message, can_send_email=False
```

Comment:
I think we expect to send emails when a suggestion gets reject, is it disabling that feature?

Reply:
I think so. There is an issue to implement this in a better way https://github.com/oppia/oppia/issues/12079. But I think we are disabling this based on this:

> translation and question suggestions probably don't need to send any feedback thread emails.

Reply:
IIRC, we internationally wanted to send emails to contributors as we don't want them to always visit the contributor-dashboard to check the status of their status.

Reply:
We are working with Claire and Cordelia to determine if this feature is actually required and if so, how important it is. I will add you to the thread.

Reply:
Also some context, this feature was broken for questions. The review flow was completely broken due to this part related to sending emails. This prevented the reviewer from accepting or rejecting any question on the server and it required a hotfix.

We also ran into an issue when submitting questions before this. It was fixed in a similar manner in https://github.com/oppia/oppia/pull/12080.